scipy/stats/sampling.py

```diff
@@ -39,7 +39,28 @@
    :toctree: generated/
 
    UNURANError
+
+
+Generators for pre-defined distributions
+========================================
+
+To easily apply the above methods for some of the continuous distributions
+in :mod:`scipy.stats`, the following functionality can be used:
+
+.. autosummary::
+   :toctree: generated/
+
+   FastGeneratorInversion
+
 """
+import math
```

Comment:
What is the reason for not using NumPy?

Reply:
Speed, I added a comment in the description of the dictionary:

> pdf: the pdf of the distribution (callable). The signature of the pdf is float -> float (i.e., the function does not have to be vectorized). If possible, functions like log or exp from the module math should be preferred over functions from numpy since the PINV setup will be faster in that case.

I think the difference is quite substantial, ~3x/4x for the setup, I tested it a while ago, I will add details later